std_msgs/Header header
float32 resolution
uint32 width
uint32 height
uint32 xorigin
uint32 yorigin
float64[] data
